codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/shapevisitor/SmithyToDafnyShapeVisitor.java

```diff
@@ -237,7 +237,7 @@ public String blobShape(final BlobShape shape) {
         dataSource,
         nilWrapIfRequired,
         dataSource,
-        someWrapIfRequired.formatted("dafny.SeqOf(v...)")
+        someWrapIfRequired.formatted("dafny.SeqFromArray(v, false)")
```

Comment:
Commenting for myself if I landed to this PR in future

`dafny.SeqOf` does not modifies v but `dafny.SeqFromArray` modifies v. However, we are fine if v is modified as v is not used after this.